src/GenAPI/Microsoft.DotNet.GenAPI.Tool/Program.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using Microsoft.DotNet.GenAPI.Shared;
+
+
+namespace Microsoft.DotNet.GenAPI.Tool
+{
+    /// <summary>
+    /// CLI frontend for the Roslyn-based GenAPI.
+    /// </summary>
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            // Global options
+            Option<string> assemblyOption = new("--assembly",
```

Comment:
Assembly is a required argument today in the existing GenAPI: https://github.com/dotnet/arcade/blob/7ab9e65d08b9a31183684781e21c3195f91ea1a8/src/Microsoft.DotNet.GenAPI/GenAPITask.cs#L42

Wouldn't you use `Argument<>` here instead of `Option<>`?